apps/provider-proxy/src/services/ProviderService/ProviderService.spec.ts

```diff
@@ -74,21 +74,6 @@ describe(ProviderService.name, () => {
         body: "hostname not allowed: test",
         expected: true
       },
-      {
-        name: "includes SSL alert number 42",
-        body: "SSL alert number 42",
-        expected: true
-      },
-      {
-        name: "includes sslv3 alert bad certificate",
-        body: "sslv3 alert bad certificate",
-        expected: true
-      },
-      {
-        name: "includes other string",
-        body: "bad certificate",
-        expected: false
-      },
```

Comment:
SSL errors are thrown at connection time. Request is not sent before handshake on TLS level. So, those tests and fix appeared to be pointless